docs/converters/tabular_converter.md

```diff
@@ -300,3 +300,36 @@ Then the following IDs will be generated / retrieved:
   `{"table": "Transformer loads", "name": "load", "key" {"Node_Number": 103, "Subnumber": 1} -> 7`
 * `sym_load.node`:
   `{"table": "Transformer loads", "name": "internal_node", "key" {"Node_Number": 103, "Subnumber": 1} -> 6`
+
+## Secure Handling
+Mapping files enable the specification of custom mappings or filter functions. These functions can come from the `power-grid-model-io` library, be user-provided, or even supplied by third parties. To ensure security, we have implemented several measures and recommend best practices to prevent malicious code execution.
+### Safe Loading of Configuration Files
+We use the `yaml.safe_load` functionality from PyYAML library to load configuration files securely. This method prevents the execution of potentially malicious code during the loading process.
+### Secure Function Handling
```

Comment:
A line between sections

Reply:
It is necessary for inserting a new line

Reply:
yes but sections typically require a new line (here and everywhere else)

```suggestion

### Secure Function Handling
```